### ISSUE TITLE
2nd Community Diligence Review of Opengate Allocator

### ISSUE THREAD

**Issue by OpenGate01** [filecoin-project/Allocator-Governance#206]
Allocator Report: 
https://compliance.allocator.tech/report/f03019921/1730161520/report.md
![image](https://github.com/user-attachments/assets/493dbb2c-064c-4a38-909c-0f9bf6e22561)
![image](https://github.com/user-attachments/assets/fd53677f-3dde-41fb-b3e7-d5543432b3f7)Compared to the report from the last review, the total retrieval success rate for nodes has doubled, and a new verified client has been added. Within the channel, four new clients have been introduced, with an additional three clients currently in communication and verification stages.

New client added this round: https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/21

This company has actual business operations and has contributed to elderly development in Australia. We had an in-person meeting with this client in late September, followed by an online session to discuss how they can distribute and back up their data, showing strong willingness to collaborate.

According to the report, their packaging aligns with the described requirements.

**Comment by filecoin-watchdog** [comment]
@OpenGate01 [Allocator Application](https://github.com/filecoin-project/notary-governance/issues/1020) 
[Compliance Report](https://compliance.allocator.tech/report/f03019921/1730374831/report.md) 
[1st Review](https://github.com/filecoin-project/Allocator-Governance/issues/128) 
1st Review score: 2.5PiB granted

2.5 PiB granted to clients:
| Client Name | DC | status |
| -------- | ------- | ------- |
| MetoldDigital | 2PiB | New |
| ANECollege |  0.5PiB | Existing |


[Example 1 - MetoldDigital #21](https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/21) 
It seems some of the discussions were held in this issue https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/14, while the actual allocation took place in issue 21. It’s a little confusing, any reason why it was split this way?

The form is completely missing information about SPs, and the only information that is known about them appeared in a comment in thread #14 on September 26, but it was not updated in thread #21, in which allocations were given. 


SPs listed in thread #14 were not used in thread #21 to make deals.
The first update of the SPs list was made in thread #21 AFTER the first allocation was made.

List of SPs updated on Oct 15 in the comment:
  "f03151449 CN,",
  "f03179555 SG,",
  "f03179572 US,",
  "f03178077 Japan  ",
  "f02826588 China"

List of SPs used for deals:
  "F03151449",
  "f03179555",
  "f03179572",
  "f03178077",
  "f02826588",
  "f03081958",
  "f02826007"

Last two SPs weren’t updated by the client.

5 of the 7 SPs have retrieval ranging from 50-95%, 2 have retrieval below 6%. Additionally, one of the SPs with low retrieval shows duplicates in the amount of 25%

The Allocator granted 2 PiBs in 3 allocations. How does that relate to the allocation schedule?

[Example 2 - ANECollege #2](https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/2) 

4 out of 9 SPs have zero retrieval, the rest do not exceed 75%.
1 of SPs has 20% duplication. 

---
Generally, the allocator does not ask too many additional questions, probably performs a KYC process (gov team should follow up on this), and uses CID reports sporadically.

**Comment by OpenGate01** [comment]
Hello,

Metolddigital Application:
The KYB verification was conducted via the link here （https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/14 ）. The comment here（https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/14#issuecomment-2378474977） shows that during our initial signing attempt, we encountered some bugs. We then asked the applicant to resubmit a new application, which was subsequently posted publicly.

⚠️ One storage provider has stored too much duplicate data - f03081958: 25.59%.
After our review, this high data redundancy was found to be due to a bug in the report, causing multiple data requests.

Our allocation plan starts with half of the weekly quota, doubling each time, with a 1PiB cap per request.

The issue with ANECollege was disclosed and reviewed in the last examination. For this allocation period, ANECollege has a lower priority than other new applicants and therefore received no allocation.

You can see that most of our applicants are reputable companies from Australia. We’ve had both online and offline discussions with Metolddigital’s leadership, so they are prioritized over other applicants.

**Comment by Kevin-FF-USA** [comment]
Hi @OpenGate01,

Thanks for the clarifications to Watchdog comments and highlights on the eduction exhibitions taking place in Australia.  Sounds like great work!

Wanted to shift our SLACK DM over to this ticket to keep you updated.  
Next steps for this refresh request will be the verifications and audit from Governance than release from the RKH.

This week the team is at the Filecoin Developer Summit, so this refill will be finalized early next.  

Warmly,

**Comment by OpenGate01** [comment]
Thank you,Australian universities are ideal for storing Filecoin cold data. They fall under the target customer category of large-scale Web2 storage, and we hope to receive more Datacap for this opportunity. We are applying for nearly 100 PiB, but so far, we have only been allocated 7.5 PiB, which would significantly aid our promotion efforts. Below is some market research data from this exhibition.                                                                                                                                                
University TB
University of Sydney 3,662
University of Melbourne 4,000+
University of New South Wales (UNSW) 3,500+
University of Queensland 4,000+
Monash University 3,800+
Australian National University (ANU) 2,500+
University of Western Australia 1,800+
University of Adelaide 1,700+
University of Technology Sydney (UTS) 1,200+
University of Newcastle 1,000+

**Comment by Kevin-FF-USA** [comment]
Hi @OpenGate01 
Wonderful!  Appreciate you sharing those metrics.  Exciting times.

Pardon the delay as this application for refresh was caught in the FDS/Holiday period.  Wanted to keep you updated, that the next step is this application for refresh will receive a final review from Galen next week. 
 
This review will take into account the retrieval rates, Bookkeeping documentation, SPs, as well as past issues that were identified in previous refreshes.  

If you have any questions please a comment here in issue, otherwise you will see guidance next week. 

Warmly,
-Kevin

**Comment by OpenGate01** [comment]
May I ask when the new datacap can be approved? Christmas is coming and many schools will have a holiday. They're all waiting for datacap. Thank you.

**Comment by galen-mcandrew** [comment]
Overall seeing mixed compliance here. While this new client has better retrieval scores, they are missing a lot of application details. 

Additionally, there is still an outstanding question around the onboarding of private encrypted data, such as education data, in a single geopolitical region. From other conversations we have had with ecosystem partners, there are restrictions on storing that data internationally from it's origin country. Can you please provide some additional details or evidence that shows how you are addressing this with these clients? What forms of KYC and KYB are you doing to verify that these clients are legitimate representatives of these universities?

From your messages, there is confusion around what type of client you are prioritizing. Are you working with public open retrievable data or private encrypted unretrievable data (Australian education data)?

Before we can make a request for more DataCap, we need clarity around the above diligence requirements for your pathway.

**Comment by OpenGate01** [comment]
@galen-mcandrew The client's original data is stored on AWS or Accelerate cloud servers. The SP (Service Provider) downloads the encrypted data processed by the client from the cloud server, and the decryption rights belong to the data owner. This complies with local data sovereignty and privacy regulations, such as the Australian Privacy Act of 1988.

**Comment by OpenGate01** [comment]
@galen-mcandrew                                                                                                                                                                    Australian universities typically have between 5 to 20 petabytes of data, making them ideal target clients for Filecoin's cold data backup services. The Australian government provides data storage subsidies to schools. However, Filecoin, as a new storage method, is not well known in Australia, and many universities do not yet recognize this form of storage. For our KYC process, we search for potential clients through various high-profile education exhibitions, conduct on-site visits to universities, and require data management leaders at the schools to undergo KYC certification. However, implementing the KYC process has been challenging in the beginning. The good news is that ANE-college has already completed the KYC certification. Everything is moving in a positive direction. We are also developing our own real data storage platform, hoping to have businesses pay for storage, which we believe is very important for Filecoin. We submitted a grant application, but it was rejected. We hope to receive more support from the Foundation.Thanks.

**Comment by OpenGate01** [comment]
@galen-mcandrew.                                                                                                                                                                                   
I have let the client to disclose their business license through GitHub. please check: https://github.com/OpenGate01/Allocator-Pathway-OpenGate01/issues/21#issuecomment-2559064237

**Comment by OpenGate01** [comment]
May I ask when our dc can be approved? Australia is almost officially at work.
Please let me know if there is anything needed, thank you.

**Comment by OpenGate01** [comment]
@galen-mcandrew.  May I ask when our dc can be approved? Australia is almost officially at work.
Please let me know if there is anything needed, thank you.

**Comment by Kevin-FF-USA** [comment]
Hi @OpenGate01, 

Thanks for submitting this application for refresh. 
Wanted to send you a friendly update - as this works its way through the system you should see a comment from Galen on behalf of the Governance this week.  If you have any questions or need support until then, please let us know.

Warmly,
-Kevin

**Comment by OpenGate01** [comment]
Hello Kevin and Galen, 
https://github.com/filecoin-project/Allocator-Governance/issues/206 
I submitted this review application in October, and I didn’t see any refill information about our application in the list https://github.com/filecoin-project/Allocator-Governance/issues/291. Could you please update it?

**Comment by Kevin-FF-USA** [comment]
Hi @OpenGate01,
This distribution and request to refresh had outliers of irregularities that has resulted in longer than average diligence reviews. Thanks for your patience, Galen will be following up again - either seeking more detailed records of the storage deals or providing next steps on decision.  *Detailed information on the KYC and documents.

**Comment by galen-mcandrew** [comment]
We have received mixed information about the requirements and data provenance for large educational institutions, and whether intercontinental distribution is permissible. Without clear guidelines, it's difficult to understand if this allocator pathway is in compliance with all the programmatic entities, such as the institutions and governments, as well as the Fil+ program.

In the hopes of supporting this project while more information is collected, we will request **2.5PiB of DataCap** through the Manual Diligence Meta Allocator. 

For the next audit, we will expect to see clear evidence of due diligence and proofs of data legitimacy.

**Comment by galen-mcandrew** [comment]
Refresh volume: 2.5PiB
Address: f2chmjasnwjwbmtbasoipel6xzhyex5deyawg2lsq // f03019921 // 0xff000000000000000000000000000000002e1491

Added to MD meta allocator, edited JSON https://github.com/filecoin-project/Allocator-Registry/pull/418
